diff --git a/node_modules/astro-bun-adapter/dist/server.js b/node_modules/astro-bun-adapter/dist/server.js
index 6a8137e..21cc163 100644
--- a/node_modules/astro-bun-adapter/dist/server.js
+++ b/node_modules/astro-bun-adapter/dist/server.js
@@ -15,25 +15,15 @@ async function sendStaticFile(pathname, localPath, options) {
   }
   return new Response(file);
 }
-function getResolvedHostForHttpServer(host) {
-  if (host === false) {
-    return "127.0.0.1";
-  } else if (host === true) {
-    return;
-  } else {
-    return host;
-  }
-}
 
 // src/server.ts
-import {App} from "astro/app";
+import { App } from "astro/app";
 function start(manifest, options) {
   const app2 = new App(manifest);
   const logger = app2.getAdapterLogger();
   const handle = handler(manifest, options);
   _server = Bun.serve({
     port: options.port,
-    hostname: getResolvedHostForHttpServer(options.host),
     fetch: handle,
     error(error) {
       return new Response(`<pre>${error}\n${error.stack}</pre>`, {
@@ -43,7 +33,7 @@ function start(manifest, options) {
       });
     }
   });
-  logger.info(`Bun server listening on port ${_server.port}`);
+  logger.info(`Server listening on ${_server.url}`);
 }
 function createExports(manifest, options) {
   const handle = handler(manifest, options);
@@ -63,7 +53,7 @@ function createExports(manifest, options) {
     handle
   };
 }
-var handler = function(manifest, options) {
+var handler = function (manifest, options) {
   const clientRoot = options.client ?? new URL("../client/", import.meta.url).href;
   const app2 = new App(manifest);
   return async (req) => {
